test/IDE/newtype.swift

```diff
@@ -129,3 +175,9 @@ func testConformances(ed: ErrorDomain) {
   acceptComparable(ed)
   acceptObjectiveCBridgeable(ed)
 }
+
+func testFixit() {
+	let _ = NSMyContextName
+	  // expected-error@-1{{'NSMyContextName' has been renamed to 'NSSomeContext.Name.myContextName'}}
```

Comment:
Can you test that the Fix-It matches as well? (with {{16-30=NSSomeContext.Name.myContextName}} or whatever)